src/core/server/base.js

```diff
@@ -13,19 +13,21 @@ import { match } from 'react-router';
 import { ReduxAsyncConnect, loadOnServer } from 'redux-async-connect';
 
 import WebpackIsomorphicTools from 'webpack-isomorphic-tools';
-import WebpackIsomorphicToolsConfig from 'config/webpack-isomorphic-tools';
+import WebpackIsomorphicToolsConfig from 'webpack-isomorphic-tools-config';
 
 import config from 'config';
 import { setJWT } from 'core/actions';
 
+const env = config.util.getEnv('NODE_ENV');
+const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
```

Comment:
I'm not sure if this is possible but maybe this might be better as `wasBuilt`. So using `npm run dev:search` it would be `false` and `npm build:search && npm start:search` would be `true` even though it is still development.